### PR TITLE
New Readme and Contributing file

### PR DESCRIPTION
**Pull request by VetleWoie**
resolves [#166](https://github.com/td-org-uit-no/tdctl-frontend/issues/166)

Adding a proper README and CONTRIBUTING file to the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,115 @@
...
+
+This is the repositroy for the backend of our website, if you want contribute to our frontend instead please look at this [repository](https://github.com/td-org-uit-no/tdctl-frontend) in stead.
+
+
+
+## Tech Stack
+
+We have chosen a tech stack that should make it easy for new students to quickly learn the ropes and start contributing.
+
+**Client:** Docker, React
```
Comment (FredrikMorstad) [inline README.md:16]: Can be moved to frontend read me

**Review by FredrikMorstad — APPROVED** [review]
Finally a descriptive read me🙌
Left one comment, but I will approve so it can be merged when completed👍

File: README.md [hunk 2]
```diff
@@ -1,49 +1,115 @@
 
-# tdctl-api
-> Application Programming Interface for TDCTL
+![Logo](https://raw.githubusercontent.com/td-org-uit-no/graphics/9fa70bc36f3d47e23f0961fe9dd5f1d0675db5a2/logo/logo-with-tagline/td-dark-tagline.svg)
 
+# Website API
 
+Welcome to the TD API project! This open-source project provides a flexible and easy-to-use interface for accessing and managing data related to TD. With this API, you can access a variety of information, including events, news, resources, and more. We believe in the power of open-source software and welcome contributions from developers of all skill levels. Join us in building a better future for TD!
+
+This is the repositroy for the backend of our website, if you want contribute to our frontend instead please look at this [repository](https://github.com/td-org-uit-no/tdctl-frontend) in stead.
```
Comment (seblj) [inline README.md:8]: repository*

File: README.md [hunk 3]
```diff
@@ -1,49 +1,115 @@
...
-
-  - **Requirements:**
-    - tdctl_api container must be running
-    - Docker SDK for python must be installed `pip install docker`
+
+Contributions are always welcome!
+
+See [`contributing.md`](./CONTRIBUTING.md) for ways to get started.
+
+Please adhere to this project's [`code of conduct`](./CODEOFCONDUCT.md).
```
Comment (seblj) [inline README.md:109]: I'll leave a comment here as well so it's easier to remember, but we should either remove this or add a CODEOFCONDUCT.md file here

**Review by seblj — APPROVED** [review]
Only one typo and a reminder of what we discussed yesterday. Looks really good!🎉

**Pull request merged**